test/integration/daemonset/daemonset_test.go

```diff
@@ -88,14 +88,14 @@ func setupScheduler(
 	cs clientset.Interface,
 	informerFactory informers.SharedInformerFactory,
 	stopCh chan struct{},
-) {
+) func() {
```

Comment:
Please add a comment explaining that the returned func() is to for restoring the plugin factory state. Or alternatively name the return func.

Reply:
+1

Reply:
Updated.